src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -101,6 +107,11 @@ public function getAddress()
         return $this->hostname.':'.$this->port;
     }
 
+    public function isXdebugUsed()
```

Comment:
I'm not sure adding this publilc method makes sense.
Let's use `extension_loaded` inline instead, don't you think?

Reply:
If we inline it, the developers could no longer disable this feature if wanted. Ok for you ?

Reply:
Nobody is going to extend this class anyway.